trie/utils/verkle.go

```diff
@@ -279,22 +275,27 @@ func GetTreeKeyStorageSlotWithEvaluatedAddress(evaluated *verkle.Point, storageK
 }
 
 func GetTreeKeyStorageSlotTreeIndexes(storageKey []byte) (*uint256.Int, byte) {
```

Comment:
That function no longer exists after the shapella rebase/your new access witness, would it make sense to try to adapt `GetTreeKeyWithEvaluatedAddess` with this technique?

Reply:
Uhm, not sure I follow. `GetTreeKeyStorageSlotTreeIndexes` is used by `GetTreeKeyStorageSlotWithEvaluatedAddress` (L272 in this file; prev function); and this last function is used every time we need to get the key for a storage slot in the trie.